### PR TITLE
Added scripts that create and open blank Swift Playground project

### PR DESCRIPTION
**Pull request by qeude**
## Description

Scripts that use [playground cli](https://github.com/JohnSundell/Playground) to create and open new Playground project.
The first script opens a basic Playground.
The second one opens a view prototyping Playground.

## Type of change

- [x] New script command

## Screenshot

basic

view

## Dependencies / Requirements

Requiring installing [playground cli](https://github.com/JohnSundell/Playground) before.

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/open-new-playground.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# Dependency: This script require playground (https://github.com/JohnSundell/Playground).
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Open new basic Playground
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon 🕊
+# @raycast.argument1 { "type": "text", "placeholder": "Destination Path (eg: ~/MyPlayground)", "optional": true }
```
Comment (unnamedd) [inline commands/developer-utils/open-new-playground.py:12]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Destination Path)", "optional": true }
```

File: commands/developer-utils/open-new-playground.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# @raycast.argument2 { "type": "text", "placeholder": "Gist, a GitHub URL or any other URL to open in Playground (eg: https://gist.github.com/JohnSundell/b7f901e8edb89d1396ede4d8db3e8c21)", "optional": true }
```
Comment (unnamedd) [inline commands/developer-utils/open-new-playground.py:13]: Huge descriptions won't affect the width of the field and we should avoid them as much as we can.

```suggestion
# @raycast.argument2 { "type": "text", "placeholder": "GitHub/Gist URL", "optional": true }
```

File: commands/developer-utils/open-new-playground.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+# @raycast.argument3 { "type": "text", "placeholder": "Add some dependencies to your playground (eg: ~/unbox/unbox.xcodeproj,~/files/files.xcodeproj) ", "optional": true }
```
Comment (unnamedd) [inline commands/developer-utils/open-new-playground.py:14]: As this placeholder looks something very important to have, I'd suggest you to right a small README containing some explanations about all of them. Check it out here: https://github.com/raycast/script-commands/tree/master/commands/apps/things

```suggestion
# @raycast.argument3 { "type": "text", "placeholder": "Dependencies", "optional": true }
```

File: commands/developer-utils/open-new-playground.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+# @raycast.packageName Developer Utils
```
Comment (unnamedd) [inline commands/developer-utils/open-new-playground.py:16]:
```suggestion
# @raycast.packageName Swift Playgrounds
```

File: commands/developer-utils/open-new-playground.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Dependency: This script require playground (https://github.com/JohnSundell/Playground).
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Open new basic Playground
```
Comment (unnamedd) [inline commands/developer-utils/open-new-playground.py:7]:
```suggestion
# @raycast.title Create Basic
```

File: commands/developer-utils/open-new-view-playground.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Open new view based Playground
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon 🕊
+# @raycast.argument1 { "type": "text", "placeholder": "Destination Path (eg: ~/MyPlayground)", "optional": true }
+# @raycast.argument2 { "type": "text", "placeholder": "Gist, a GitHub URL or any other URL to open in Playground (eg: https://gist.github.com/JohnSundell/b7f901e8edb89d1396ede4d8db3e8c21)", "optional": true }
+# @raycast.argument3 { "type": "text", "placeholder": "Add some dependencies to your playground (eg: ~/unbox/unbox.xcodeproj,~/files/files.xcodeproj) ", "optional": true }
```
Comment (unnamedd) [inline commands/developer-utils/open-new-view-playground.py:14]: The same explanation about huge placeholders I made above, applies here.

File: commands/developer-utils/open-new-view-playground.py [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+# @raycast.packageName Developer Utils
```
Comment (unnamedd) [inline commands/developer-utils/open-new-view-playground.py:16]:
```suggestion
# @raycast.packageName Swift Playgrounds
```

File: commands/developer-utils/open-new-view-playground.py [hunk 8]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Dependency: This script require playground (https://github.com/JohnSundell/Playground).
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Open new view based Playground
```
Comment (unnamedd) [inline commands/developer-utils/open-new-view-playground.py:7]:
```suggestion
# @raycast.title Create View Based
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Nice addition this pull request @qeude! 

However, I strongly suggest you to move it to a new folder, maybe called `swift-playground` or just `playground` inside of `apps` folder. As you have more than one Script Command, I do think it will fit better there, even though it's a developer utility.

**Comment by qeude** [comment]
@unnamedd Thanks for those feedbacks, I did make the changes 😄

**Comment by fe9lix** [comment]
Hey @qeude! I think this got by mistake into the root folder in `commands` now. The idea was to move it into the apps folder, correct?

**Comment by qeude** [comment]
@fe9lix My bad, thought the idea was to put the playground folder in the `commands`, I'll fix this

**Review by fe9lix — APPROVED** [review]

**Pull request merged**